### PR TITLE
Adding docs around the Subnet Available collector and analyzer

### PR DESCRIPTION
**Pull request by CpuID**
Self explanatory.

Should be merged with a release of https://github.com/replicatedhq/troubleshoot/pull/1004 (with the version number it was introduced?)

### CODE REVIEW

**Comment by paigecalvert** [comment]
Holding on the docs review until Evans has a chance to review, but wanted to ask if this new topic might also need to be manually added to the TOC? Asking because I did not see it in the TOC in the Netlify preview (I think it's supposed to be in this section): https://deploy-preview-473--troubleshoot-sh.netlify.app/docs/host-collect-analyze/overview/

**Review by banjoh — APPROVED** [review]
Content LGTM. We'll want to wait for the feature to be released first before merging though

**Comment by jonquil2002** [comment]
@CpuID Looks good! I just made some minor edits to your content to reflect our style guide.

Two things I think are missing:

- It looks like it should be listed and linked to from the All Host Collectors page. https://deploy-preview-473--troubleshoot-sh.netlify.app/docs/host-collect-analyze/all/
- It seems to be missing from the TOC on the left. It should be added alphabetically, though there is at least one mistake in that TOC.

Lmk when you're ready for a final review.

File: docs/source/host-collect-analyze/subnetAvailable.md
Comment (banjoh) [inline docs/source/host-collect-analyze/subnetAvailable.md:34]: Please update this example in light of [this comment](https://github.com/replicatedhq/troubleshoot/pull/1004#pullrequestreview-1305738327)

File: docs/source/host-collect-analyze/subnetAvailable.md
Comment (banjoh) [inline docs/source/host-collect-analyze/subnetAvailable.md:73]: Please update this example in light of [this comment](https://github.com/replicatedhq/troubleshoot/pull/1004#pullrequestreview-1305738327)

File: docs/source/host-collect-analyze/subnetAvailable.md
Comment (banjoh) [inline docs/source/host-collect-analyze/subnetAvailable.md:74]: Please update this example in light of [this comment](https://github.com/replicatedhq/troubleshoot/pull/1004#pullrequestreview-1305738327)

**Review by banjoh — CHANGES_REQUESTED** [review]

**Comment by CpuID** [comment]
> I just made some minor edits to your content to reflect our style guide.

thanks @jonquil2002 :)

**Comment by CpuID** [comment]
I'll sort out getting this on the TOC, I missed that one 👍

**Review by banjoh — APPROVED** [review]

**Comment by jonquil2002** [comment]
@CpuID @banjoh lmk when you're ready for a final docs review! :)

**Comment by CpuID** [comment]
Sorted out the TOC + a few other outstanding items mentioned above, thanks @jonquil2002 @paigecalvert 

This is ready to merge once https://github.com/replicatedhq/troubleshoot/pull/1004 is released 👍

**Review by jonquil2002 — APPROVED** [review]
lgtm

**Review by banjoh — APPROVED** [review]
All good!

**Comment by CpuID** [comment]
heads up: https://github.com/replicatedhq/troubleshoot/pull/1004 is merged but not released yet

holding on a merge of this PR until it's released

**Comment by CpuID** [comment]
Released, 0.59.0

**Pull request merged**
